.travis.yml

```diff
@@ -0,0 +1,32 @@
+language: elixir
+elixir:
+  - '1.8.1'
+otp_release:
+  - '21.3'
+services:
```

Comment:
I don't think we need this docker stuff for this repo

Reply:
It has integration tests so I believe that we do.

Reply:
Oops, there are integration tests here.